app/views/miq_request/_prov_vm_grid.html.haml

```diff
@@ -1,3 +1,3 @@
 #prov_vm_div
-  - initial_data = prov_grid_data(@edit, @vms, @vm)
+  - initial_data = prov_vm_grid_data(@edit, @vms, @vm)
```

Comment:
Similar here we don't need the intermediate variable (If you change in one place might as well change int he other)